Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Use an official Python runtime as a parent image
+FROM python:3.9-slim-buster
+
+# Install Git
+RUN apt-get update && \
+    apt-get install -y git && \
+    rm -rf /var/lib/apt/lists/*
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the pyproject.toml and other necessary files
+COPY ./pyproject.toml /app/
+COPY ./README.md /app/
+COPY ./LICENSE /app/
+
+# Copy the current directory contents into the container at /app
+COPY . /app/
+
+# Copy the .git directory for versioning
+COPY .git /app/.git/
+
+# Install Flit and other build dependencies
+RUN pip install flit
```

Comment:
Is flit even required for this? I know it's in the pyproject file but it can maybe be removed since it's not actually used for building a wheel or anything.